index.html

```diff
@@ -108,12 +78,11 @@ <h4 class="h4">Headless Rendering</h4>
         <div>
           <h2 class="h2">Work smarter with nodes</h2>
           <p class="p-body">
-            No more frustrating hunts through nested precomps! Fusion and Nuke users will be right at home in Natron’s node graph. Group, merge, and tinker to achieve just about any effect. Create pixel perfect mattes, cleanplates, and set
-            extensions, all with node trees.
+            No more hunting through nested pre-comps!
+            Nuke and Fusion users will be right at home with Natron's node-based workflow. Group, merge, and tinker to achieve just about any effect. Create pixel perfect mattes, set-extensions, Hollywood-style VFX and more, all with Natron's offerings.
```

Comment:
Don't love the "Hollywood-style VFX" addition, mostly because Natron is seldom used (if ever) in North American VFX shops.

```suggestion
            Nuke and Fusion users will be right at home with Natron's node-based workflow. Group, merge, and tinker to achieve just about any effect. Create pixel perfect mattes, set-extensions, and more!
```